docs/leo-rover/integrations/lidars/ouster-os0.mdx

```diff
@@ -0,0 +1,406 @@
+---
+title: Ouster LiDAR
+sidebar_label: Ouster LiDAR
+keywords:
+  - ouster
+  - os0
+  - lidar
+  - 3d
+  - leo
+  - integration
+  - waterproof
+  - example
+description: >
+  Detailed tutorial on setting up the Ouster LiDAR, a powerful and lightweight
+  sensor, that works in both outdoor and indoor autonomous  vehicles and mobile
+  robots.
+
+image: /img/robots/leo/integrations/ouster-os0/ouster-os0.webp
+---
+
+This tutorial will guide you through the process of connecting and using Ouster
+LiDAR sensor on your rover. This tutorial will use Ouster OS0, however the
+integration process is the same for other models such as OS1 and OS2.
+
+Light Detection and Ranging devices, or Lidars for short, are mechanisms used
+for mapping the environment, object detection, tracking the speed of vehicles
+and in a wide range of other applications. Ouster devices excel at 3D scanning
+near-range distances within dense, cluttered environments. The OS2 has a range
+of 100 m, and the sensor's light weight (522 g), compact footprint (~Ø87 mm x 58
+mm), and 90 degree vertical field of view make it ideal not only for autonomous
+vehicles but also robotics and mobile 3D mapping applications.
+
+## What to expect?
+
+After finishing the tutorial you will have configured the Ouster LiDAR to work
+on your rover. You will be able to gather and visualize the data from the
+sensor.
+
+## Prerequisites
+
+<LinkButton docId="leo-rover/guides/ssh" />
+<LinkButton docId="leo-rover/guides/connect-to-network" />
+<LinkButton docId="leo-rover/advanced-guides/install-ros-on-your-computer" />
+<LinkButton docId="leo-rover/advanced-guides/ros-development" />
+
+## Mechanical integration
+
```

Comment:
Please avoid empty sections. For users, please add something like:

_Work in progress_

The empty section can be mistaken as an oversight.